deps.ts

```diff
@@ -1,22 +1,6 @@
-// std/
-export {
-  getCookies,
-  setCookie,
-} from "https://deno.land/std@0.152.0/http/cookie.ts";
-
 // x/
-export * from "https://deno.land/x/fresh@1.0.2/server.ts";
-export * from "https://deno.land/x/fresh@1.0.2/runtime.ts";
 export { CSS } from "https://deno.land/x/gfm@0.1.22/mod.ts";
 
-// esm.sh/
-export * from "https://esm.sh/preact@10.10.0";
-export { renderToString } from "https://esm.sh/preact-render-to-string@5.2.1?external=preact";
-export { type Configuration, setup, tw } from "https://esm.sh/twind@0.16.17";
-export { virtualSheet } from "https://esm.sh/twind@0.16.17/sheets";
-export { default as Prism } from "https://esm.sh/prismjs@1.25.0?pin=v55";
```

Comment:
Prism must be exported from here (for the other prism related imports to find the "Prism" thingy). Otherwise, example pages fail to render: https://yex-zwjx2b7ekb0g.deno.dev/hello-world